.travis.yml

```diff
@@ -14,3 +14,15 @@ script:
 - "./debuild.sh |& tee log.txt"
 notifications: 
 slack: fossasia:3pMkQrSogvEMfnu9aJDW2ZYC
+before_deploy:
+- export lock_package="`echo systemlock_*_all.deb`"
+- cat systemlock_*_all.deb > "/home/travis/${lock_package}"
+- cp log.txt /home/travis/log.txt
+deploy:
+  provider: releases
+  api_key:
+    secure: 
```

Comment:
We need a secure key

Reply:
We need a key from one of the project maintainers

Reply:
If you could Travis dashboard it, instead of a secure key, I could provide one

Reply:
@abishekvashok I have removed the security_key now we can add the github oauth token for the meilix-systemlock project